static/src/stylesheets/module/content/_live-blog.scss

```diff
@@ -478,12 +478,11 @@ $block-padding-right: $gs-gutter;
 .dropdown--live-feed {
     border-top: 0;
 
+    $content-gutter: $gs-gutter / 2;
```

Comment:
Thinking about it - variable setting is such a small, if even, and abstraction, more of convenience. Agreed with the generic naming, which could lead to someone using it out of scope.